tests/unit/spanner_dbapi/test_connection.py

```diff
@@ -383,14 +383,14 @@ def test_run_statement_dont_remember_retried_statements(self):
 
         self.assertEqual(len(connection._statements), 0)
 
-    def test_run_statement_w_heterogenous_insert_statements(self):
-        """Check that Connection executed heterogenous insert statements."""
+    def test_run_statement_w_homogeneous_insert_statements(self):
```

Comment:
It looks like you overwrote `test_run_statement_w_heterogenous_insert_statements` during the conflict resolution. Please add it back in :+1: